src/status_im/constants.cljs

```diff
@@ -53,7 +53,7 @@
 (def ^:const mute-for-15-mins-type 1)
 (def ^:const mute-for-1-hour-type 2)
 (def ^:const mute-for-8-hours-type 3)
-(def ^:const mute-for-1-week 4)
+(def ^:const mute-for-1-week 6)
```

Comment:
Replaced the 1-week timer with a 1-minute timer for testing purposes. (Should be reverted before merging)